src/SFA.DAS.AssessorService.Web.Staff/Views/CertificateAmend/Uln.cshtml

```diff
@@ -33,29 +40,29 @@
-                        <label class="form-label" for="Uln">
-                            <span class="form-label-bold">Unique learner number (<abbr title="Unique learner number">ULN</abbr>)</span>
-                            <span class="form-hint">"For example, 1234567890"</span>
-                            <span class="error-message" asp-validation-for="Uln"></span>
-                        </label>
-                        <input class="form-control js-integers-only" name="Uln" autocomplete="off" asp-for="Uln" type="text" sfa-validation-for="Uln" sfa-validationerror-class="form-control-error" maxlength="10" />
-                    </div>
+                <div class="form-group" sfa-validation-for="Uln" sfa-validationerror-class="form-group-error">
+                    <label class="form-label" for="Uln">
+                        <span class="form-label-bold">Unique learner number (<abbr title="Unique learner number">ULN</abbr>)</span>
+                        <span class="form-hint">"For example, 1234567890"</span>
+                        <span class="error-message" asp-validation-for="Uln"></span>
+                    </label>
+                    <input class="form-control js-integers-only" name="Uln" autocomplete="off" asp-for="Uln" type="text" sfa-validation-for="Uln" sfa-validationerror-class="form-control-error" maxlength="10" />
+                </div>
 
-                    <div class="form-group" sfa-validation-for="ReasonForChange" sfa-validationerror-class="form-group-error">
-                        <label for="ReasonForChange">
-                            <span class="form-label-bold">Reason for change</span>
-                            <span class="form-hint">Please could you provide more detail as to why this change is required.</span>
-                            <span class="error-message" asp-validation-for="ReasonForChange"></span>
-                        </label>
-                        <textarea class="form-control form-control-3-4" rows="5" name="ReasonForChange" sfa-validation-for="ReasonForChange" sfa-validationerror-class="form-control-error" asp-for="ReasonForChange"></textarea>
-                    </div>
+                <div class="form-group" sfa-validation-for="ReasonForChange" sfa-validationerror-class="form-group-error">
+                    <label for="ReasonForChange">
+                        <span class="form-label-bold">Reason for change</span>
+                        <span class="form-hint">Please could you provide more detail as to why this change is required.</span>
+                        <span class="error-message" asp-validation-for="ReasonForChange"></span>
+                    </label>
+                    <textarea class="form-control form-control-3-4" rows="5" name="ReasonForChange" sfa-validation-for="ReasonForChange" sfa-validationerror-class="form-control-error" asp-for="ReasonForChange"></textarea>
```

Comment:
Seems to be a lot of the same markup for Reason for change in many cshtmls.  Can this be refactored out into a partial view?

Reply:
This was existing code that I reused. All views in assessor needs refactoring this should be and seperate exercise.

Reply:
I did ask about this screen and I was told it has been already implemented just reuse what is already there and that is what I did. This code already had been reviewed and accepted.